server/services/yeelight/lib/device/discover.js

```diff
@@ -0,0 +1,66 @@
+const logger = require('../../../../utils/logger');
+const models = require('../models');
+const { DEVICE_EXTERNAL_ID_BASE, DEVICES_MODELS, COMMAND_TYPE } = require('../utils/constants');
+
+/**
+ * @description Send a broadcast to find the devices
+ * @returns {Promise<Array>} Resolve with array of new devices.
+ * @example
+ * discover();
+ */
+async function discover() {
+  const discovery = new this.yeelightApi.Discover();
```

Comment:
Still not fixed, here if the user go to the "discovery" tab and leave the tab, and come back to the tab, it throws an `EADDRINUSE` error: 

<img width="869" alt="Screenshot 2020-11-20 at 16 20 23" src="https://user-images.githubusercontent.com/7365207/99810453-63fd8500-2b4c-11eb-905c-fc770f72b968.png">

This discover process should be asynchronous, and add devices to a list in RAM which could be queried in a different route that the discover one + sent in websockets.

This is what we do in the Z-Wave integration for example :) 

We can talk more about it if you have questions about it.